templates/advantage/index.html

```diff
@@ -195,6 +195,7 @@ <h2>Your free personal subscription</h2>
 
   <div class="row">
     <div class="col-12 u-hide--small">
+      <p>Anyone can use UA Infrastructure Essential for free on up to 3 machines (limitations apply). All you need is an Ubuntu One account. And if you’re a recognised <a href="https://wiki.ubuntu.com/Membership" class="p-link--external">Ubuntu community member</a>, it’s free on up to 50 machines.</p>
```

Comment:
```suggestion
      <p>Anyone can use UA Infrastructure Essential for free on up to 3 machines (limitations apply). And if you’re a recognised <a href="https://wiki.ubuntu.com/Membership" class="p-link--external">Ubuntu community member</a>, it’s free on up to 50 machines.</p>
```

Sorry I overlooked this bit. No point in telling you that you need a U1 account when you’re already signed in.